### DISCUSSION TITLE
[Backend] New Laravel implementation

### DISCUSSION THREAD

**Discussion by f1amy** [realworld-apps/realworld#703] [Added implementations]
Using docker-compose, GitHub Actions CI (static analysis and tests), Swagger UI
Currently working on tests, and completing API documentation
https://github.com/f1amy/laravel-realworld-example-app

**Comment by f1amy** [comment]
My implementation is now complete, please check it out!
@geromegrignon

**Reply by geromegrignon** [reply]
Approved!
Thank so much for your contribution
